docker-compose.halfstack-ha.yml

```diff
@@ -3,107 +3,128 @@ version: "2.4"
 services:
 
   backendai-half-db:
-    image: postgres:13.7-alpine
+    image: postgres:15.1-alpine
     command: postgres -c 'max_connections=256'
     networks:
       - half
     ports:
-      - "8200:5432"
+      - "8100:5432"
```

Comment:
This should be consistent with `docker-compose.halfstack.main.yml` and the default value of `POSTGRES_PORT`.